community/modules/compute/schedmd-slurm-gcp-v5-node-group/variables.tf

```diff
@@ -304,6 +304,17 @@ variable "reservation_name" {
   nullable    = false
 }
 
+variable "maintenance_interval" {
+  description = "Specifies the frequency of planned maintenance events. The accepted values are: PERIODIC"
```

Comment:
In the description I would do somethimg more like:

```
The only accepted value is \"PERIODIC\". Leave omitted to avoid specifying an interval.
```